dashboards/erlang/main.json

```
@@ -167,7 +167,7 @@
               },
               {
                 "key": "type",
-                "value": "scheduler"
+                "value": "normal"
```

```suggestion
                "value": "*"
```

If we're planning to add more things in the future, we can also make it a wildcard?

---

Should those other schedulers be shown on the same graph? I think it might be confusing to see utilization of CPU schedulers, IO schedulers and Erlang schedulers all in one graph.

---

Ah that kind of different schedulers? Then this is fine.